### PR TITLE
Add prevfloat(x, n) for convenience.

### PR DESCRIPTION
**Pull request by Ellipse0934**
Just like nextfloat(x,n).

### CODE REVIEW

**Comment by dpsanders** [comment]
You'll need to add some tests.

File: base/float.jl [hunk 1]
```diff
@@ -637,6 +637,15 @@ such `y` exists (e.g. if `x` is `Inf` or `NaN`), then return `x`.
 """
 nextfloat(x::AbstractFloat) = nextfloat(x,1)
 
+"""
+    prevfloat(x::AbstractFloat, n::Integer)
+
+The result of `n` iterative applications of `prevfloat` to `x` if `n >= 0`, or `-n`
+applications of `nextfloat` if `n < 0`.
+"""
+
```
Comment (ararslan) [inline base/float.jl]: This empty line between the docstring and the function signature should be removed.

**Comment by stevengj** [comment]
You'll also need to add it to the `exports.jl` file, and link to it from `doc/src/base/numbers.md`

File: base/float.jl [hunk 2]
```diff
@@ -637,6 +637,14 @@ such `y` exists (e.g. if `x` is `Inf` or `NaN`), then return `x`.
 """
 nextfloat(x::AbstractFloat) = nextfloat(x,1)
 
+"""
+    prevfloat(x::AbstractFloat, n::Integer)
+
+The result of `n` iterative applications of `prevfloat` to `x` if `n >= 0`, or `-n`
+applications of `nextfloat` if `n < 0`.
```
Comment (stevengj) [inline base/float.jl]: Add cross-reference ``` [`nextfloat`](@ref) ```

Also maybe say ``` Equivalent to `nextfloat(x, -n)`. ```

**Comment by mbauman** [comment]
This function is already exported and linked in the documentation as we already provide the one-argument form, so no need to change `exports` or `doc/src/base/numbers.md`.

**Comment by ararslan** [comment]
@Ellipse0934, can you elaborate on why you've closed this?

**Comment by Ellipse0934** [comment]
`prevfloat(x, n)` has been implemented in 1.0 .
```
julia> prevfloat(1.0, 15)
0.9999999999999983

help?> prevfloat
search: prevfloat

  prevfloat(x::AbstractFloat, n::Integer)

  The result of n iterative applications of prevfloat to x if n >= 0, or -n
  applications of nextfloat if n < 0.

  ────────────────────────────────────────────────────────────────────────────

  prevfloat(x::AbstractFloat)

  Return the largest floating point number y of the same type as x such y < x.
  If no such y exists (e.g. if x is -Inf or NaN), then return x.
```

**Comment by ararslan** [comment]
Ah, I see. Thanks.

**Pull request closed without merging**